deploy/deploy.go

```diff
@@ -71,6 +72,28 @@ func logCommand(cmd string, args ...string) error {
 	return c.Run()
 }
 
+// outLogCommand runs the specified command but records standard output
+// with log.Info and standard error with log.Warning. Standard output
```

Comment:
Since standard errors are being recorded, should the log level be log.Error instead of Warning?

Reply:
Sometimes CLIs write to stderr even when there is not actually an error that we would need to concern KNE users with. If the cmd fails the user should get an error message from KNE with the relevant bits of the stderr/out included. I believe that warning is more appropriate in this case

Reply:
Thanks for the clarification!